.github/workflows/test-push-workflows-service-image.yml

```
@@ -174,171 +156,15 @@ jobs:
       environment: 'dev'
-      tag: ${{ needs.build-and-push-ee-image.outputs.docker_tag }}
-    secrets: inherit
-
-  run-migrations:
-    needs: [build-and-push-ee-image,determine_branch]
-    if: ${{(github.event_name == 'repository_dispatch') || (github.event_name == 'workflow_dispatch' && inputs.operation == 'Run Migrations')}}
-    uses: ./.github/workflows/test-db-ops.yaml
-    with:
-      environment: ${{ needs.determine_branch.outputs.branch }}
-      tag: ${{ needs.build-and-push-ee-image.outputs.docker_tag }}
+      sha: ${{ needs.build-and-push-image.outputs.sha_short }}
```

actionlint

input "sha" is not defined in "./.github/workflows/test-deploy-wf-service.yml" reusable workflow. defined input is "environment"

(workflow-call)